integrations/helm/release/release.go

```diff
@@ -296,3 +298,49 @@ func mergeValues(dest, src chartutil.Values) chartutil.Values {
 	}
 	return dest
 }
+
+func releaseManifestToUnstructured(manifest string, logger log.Logger) []unstructured.Unstructured {
+	manifests := helmutil.SplitManifests(manifest)
+	var objs []unstructured.Unstructured
+	for _, manifest := range manifests {
+		json, err := yaml.YAMLToJSON([]byte(manifest))
+		if err != nil {
+			logger.Log("err", err)
+			continue
+		}
+
+		var u unstructured.Unstructured
+		if err := u.UnmarshalJSON(json); err != nil {
```

Comment:
This made me pause for thought -- the usual formulation is

```
var u unstructured.Unstructured
if err := json.Unmarshal(bytes, &u); err != nil {
 ...
}
```

.. though this code will work fine, so far as I know (it's just eliding the dispatch which json.Unmarshal would do after checking `&u` implemented `json.Unmarshaler`)

Reply:
The reason I choose for `u.UnmarshalJSON` and not the common `json.Unmarshal` is that it makes use of `UnstructuredJSONScheme`, which to my knowledge does some sanity checks; it will for example fail if there is no `kind` key set.

This may however be over engineered and I am happy to change it to a simpler approach.

Reply:
Additional checks are good :+1: `json.Unmarshal` will end up calling `UnmarshalJSON` for anything that implements it, so you will still get those.